dist/RollbarSourceMapPlugin.js

```diff
@@ -0,0 +1,191 @@
+'use strict';
```

Comment:
Can you remove the dist dir from the commits? We don't want the dist directory to be in source control. The build will run and CI and that will be published to npm.